pkg/reconciler/v1alpha1/route/reconcile_resources.go

```diff
@@ -98,8 +99,10 @@ func (c *Reconciler) reconcilePlaceholderService(ctx context.Context, route *v1a
 			}
 		}
 	}
-	// Update the route's domain internal field
-	route.Status.DomainInternal = resourcenames.K8sServiceFullname(route)
+	// Update the information that makes us Targetable.
```

Comment:
I think it might be best to keep setting the route.Status.DomainInternal for some time? To allow things to coexist for awhile?

Reply:
Done